php/libraries/Project.class.inc

```diff
@@ -1,9 +1,9 @@
-<?php
+<?php declare(strict_types=1);
 /**
  * This file contains a class which encapsulates the concept of a "Project"
  * in LORIS.
  *
- * PHP Version 5
+ * PHP Version 7
```

Comment:
Shouldn't it be 7.2?

Reply:
that is the minimum version of PHP require to run the code properly. Type declaration require php7.0.